community/contribute/doc-files-structure-guide-md

```diff
@@ -0,0 +1,37 @@
+# Documentation Files Structure
+
+## Structure Best Practice
+
+### Overview - mandatory
+
+- What is it, what do I need it for? Benefits?
+- What organization/company does this belong to? (Link to Osism main documentation?)
```

Comment:
Because this is a styleguide for all contributing companies, I would like to delete the Osism and fill it with company. :)